e2e/README.md

```diff
@@ -14,18 +14,12 @@ You run the tests on example apps with `yarn test:e2e` (or `npm run test:e2e)`.
 
 What will happen in order is as following:
 
-1. A bundle (we'll call it `[bundle]`) will be created for `jest-preset-angular` using `npm pack` (`yarn pack` is buggy).
-
-    The `prepare` script will be run, so `clean` and `build` (that is why e2e tests are launched before the others, since it's building as part of the process)
- 
-2. `npm ci` and `npm install --save-dev [bundle]` will be run in each project directory in `e2e` directory.
-3. each test suite in each example project are run according to `jest` config of that project.
+1. Each project will install its dependencies
+2. Each test suite in each example project runs according to `jest` config of that project.
```

Comment:
again, better use: *in each **test** project*

Reply:
done